mne/preprocessing/maxwell.py

```diff
@@ -25,21 +26,24 @@
 from ..io.tag import find_tag, _loc_to_trans
 from ..io.pick import pick_types, pick_info, pick_channels
 from ..utils import verbose, logger
+from ..externals.six import string_types
 
 
 @verbose
-def maxwell_filter(raw, origin=(0., 0., 40.), int_order=8, ext_order=3,
+def maxwell_filter(raw, origin='default', int_order=8, ext_order=3,
                    fine_cal=None, ctc=None, st_dur=None, st_corr=0.98,
-                   verbose=None):
+                   coord_frame='head', verbose=None):
     """Apply Maxwell filter to data using spherical harmonics.
 
     Parameters
     ----------
     raw : instance of mne.io.Raw
         Data to be filtered
-    origin : array-like, shape (3,)
+    origin : array-like, shape (3,) | str
         Origin of internal and external multipolar moment space in head coords
-        and in millimeters
+        and in meters. The default is ``'default'``, which means
+        ``(0., 0., 40e-3)`` for ``coord_frame='head'`` and
+        ``(0., 13e-3, -6e-3)`` for ``coord_frame='meg'``.
```

Comment:
Where do these numbers come from? Should I be using those for device frame coords? (I've always used (0., 0., 0.))